examples/freertos_linux/Linux_gcc_gcp_iot/README.md

```diff
@@ -6,22 +6,41 @@ This example uses the Google Cloud IoT Device SDK for Embedded C to connect a Fr
 
 Follow the steps below to connect the FreeRTOS application to the MQTT bridge.
 
-Before you begin, generate a [public/private key pair](https://cloud.google.com/iot/docs/how-tos/credentials/keys), store the private key in the `examples/freertos_linux/Linux_gcc_gcp_iot` directory, and name the key `ec_private.pem`. 
+1. Go to the repository's root directory.
```

Comment:
This is an instruction. Reading it I thought I have to figure out something, although this just describes the effects of the command below. I'd just keep the style: `run this please`, `btw: this is the result`. @soyrice PTAL

Reply:
s/./: would work?